docs/advanced/repo_subdirectories.rst

```diff
@@ -0,0 +1,17 @@
+Organizing cookiecutter templates in subdirectories
```

Comment:
@insspb I see other features have for example `Dictionary Variables (1.5+)`.  Should I include this?

Reply:
Yes, my review note was mention that you should add this document to index toc.